### ISSUE TITLE
Does multi-user setup and user_type column support Larave's morphMap?

### ISSUE THREAD

**Issue by SirLamer** [santigarcor/laratrust#119]
If we can use morphMap this will reduce the length of the string saved to user_type, and reduce the size of the database as well asa aa small I/O savings.

Example from AppServiceProvider.php
```php
<?php

namespace App\Providers;

use Illuminate\Database\Eloquent\Relations\Relation;
use Illuminate\Support\ServiceProvider;
use App\Models;

class AppServiceProvider extends ServiceProvider
{
    /**
     * Bootstrap any application services.
     *
     * @return void
     */
    public function boot()
    {
        Relation::morphMap([
            Models\User\User::class,
        ]);
    }
}
```

Example concept of config setup in laratrust.php:
```php

    /*
    |--------------------------------------------------------------------------
    | Laratrust User Models
    |--------------------------------------------------------------------------
    */
    'user_models' => [
        'users' => 'user',
    ],
```

**Comment by santigarcor** [comment]
Well currently it uses just normal morph but you can try using the morph map.

**Comment by SirLamer** [comment]
I tried using the morph map but no good.

I am not especially familiar with how morph map works on the back-end in Laravel, I could have a go at a PR but I'd be approaching it from scratch.

**Comment by fntneves** [comment]
I am also interested in this enhancement.

It is actually not good to have multiple morphmaps defined around the project. (AppServiceProvider and config files)

If you need help to implement this, let me know.

**Comment by santigarcor** [comment]
@fntneves This is already implemented check 1f537031ff3b4a7754b4fdb90d229f29031b13ed.
